posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,526 @@
+---
+title: "Getting Started with the `napari-cosmx` plugin"
+author:
+  - name: Evelyn Metzger
+    affiliations: 
+      - ref: nstg
+      - ref: eveilyeverafter
+affiliations:
+  - id: nstg
+    name: NanoString Technologies, Inc.
+    city: Seattle
+    state: WA
+  - id: eveilyeverafter
+    name: "Github: [@eveilyeverafter](https://github.com/eveilyeverafter)"
+toc: true
+toc-title: Contents
+toc-depth: 3
+toc-expand: 2
+toc-location: left
+number-sections: true
+number-depth: 4
+date: "2024-04-19"
+categories: [visualization, napari]
+draft: false
+image: figures/prostate.gif
+---
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-pancreas"
+#| fig-cap: "Nine Fields of View (FOVs) of a whole-transcriptome Pancreas dataset visualized with the `napari-cosmx` plugin. DAPI and PanCK is shown in blue and green, respectively. Endocrine cells in the Islets of Langerhans can be identified by their transcript abundance of marker genes (points). Red = _GCG_ (alpha cells), orange = _INS_ (beta cells), cyan = _SST_ (delta cells)."
```

Comment:
DAPI and PanCK **are** shown

Reply:
Done. 81254407d9f9f69d537bafe6b047d2c2a4d4bb16